src/standardeditingmodeediting.js

```diff
@@ -54,5 +54,12 @@ export default class StandardEditingModeEditing extends Plugin {
 		} );
 
 		editor.commands.add( 'restrictedEditingException', new RestrictedEditingExceptionCommand( editor ) );
+
+		// Set unique class to every $root element for additional styling in standard editing
+		editor.editing.view.change( writer => {
+			for ( const root of editor.editing.view.document.roots ) {
+				writer.addClass( 'ck-standard-editing', root );
```

Comment:
```suggestion
				writer.addClass( 'ck-standard-editing-mode, root );
```

Reply:
1. There should be 2 classes to avoid using `:not()` etc.
2. Both classes should be prefixed with `ck-restricted-editing` so there's no doubt where they come from.
3. They should use the BEM [modifier syntax](https://en.bem.info/methodology/css/#modifiers).
4. Which comes down to `ck-restricted-editing_mode_standard` and `ck-restricted-editing_mode_restricted`.

Reply:
> * There should be 2 classes to avoid using `:not()` etc.

The other class is defined in the other plugin.

Reply:
Fixed 👌